src/AppBundle/Form/DataTransformer/TagArrayToStringTransformer.php

```diff
@@ -54,7 +54,7 @@ public function reverseTransform($string)
             return [];
         }
 
-        $names = explode(',', $string);
+        $names = array_filter(array_unique(array_map('trim', explode(',', $string))));
```

Comment:
Not sure about strtolower, for instance I want my "JavaScript", "WordPress" tags to keep the upcases.